pkg/api/testapi/testapi.go

```diff
@@ -78,6 +81,12 @@ func init() {
 			internalGroupVersion: extensions.SchemeGroupVersion,
 		}
 	}
+	if _, ok := Groups[batch.GroupName]; !ok {
+		Groups[batch.GroupName] = TestGroup{
+			externalGroupVersion: unversioned.GroupVersion{Group: batch.GroupName, Version: registered.GroupOrDie(batch.GroupName).GroupVersion.Version},
+			internalGroupVersion: extensions.SchemeGroupVersion,
```

Comment:
I missed this in @piosz's change-- can you move the following `if _, ok := Groups[extensions.GroupName]; !ok {` section above the batch & autoscaling setup code? otherwise, referencing the extensions group here may not do what we want.